docs/example_projectq_grover.py

```diff
@@ -111,10 +111,11 @@ def get_authentication():
     # Remote Quantum-Inspire backend #
     authentication = get_authentication()
     qi = QuantumInspireAPI(QI_URL, authentication)
+    qi_backend = QIBackend(quantum_inspire_api=qi)
 
-    compiler_engines = restrictedgateset.get_engine_list(one_qubit_gates="any", two_qubit_gates=(CNOT, CZ, Toffoli))
+    compiler_engines = restrictedgateset.get_engine_list(one_qubit_gates=qi_backend.one_qubit_gates,
```

Comment:
Given that toffoli is not a two_qubit_gate, I understand why you remove it here, but how do we add three_qubit_gates?

Reply:
It was an error (in my opinion) to add toffoli to the 2-qubit gates.
I did the following:
- Besides the one_qubit_gates and two_qubit_gates there is a possibility to add 'other gates to the restrictedgateset engine. This parameter is called other_gates. I will add the three qubit gates to this parameter (thus the toffoli). Because the old example was actually without toffoli (because it wasn't a 2 bit gate) I added only the one and two qubit gates. I will for this example add the three qubit gates also.